ElectronClient/gui/NoteEditor/NoteEditor.tsx

```diff
@@ -366,6 +366,7 @@ function NoteEditor(props: NoteEditorProps) {
 	}
 
 	const searchMarkers = useSearchMarkers(showLocalSearch, localSearchMarkerOptions, props.searches, props.selectedSearchId);
+	// searchMarkers.keywords = props.highlightedWords;
```

Comment:
Please remove commented out code from the pull request, it makes it more difficult to follow.